src/System.AppContext/tests/System.AppContext.Tests.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <ProjectGuid>{6F04B167-A03F-4206-8481-60213C3EF9B9}</ProjectGuid>
     <Configurations>netstandard-Debug;netstandard-Release</Configurations>
+    <AssemblyBeingTestedName Condition="'$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'netstandard'">System.Private.CoreLib</AssemblyBeingTestedName>
```

Comment:
This is pretty long condition that is not easy to understand. Also, we will likely want to tweak it overtime (e.g. to include uap or aot).

Would it be better to have a simple property for this instead of replicating this condition everywhere? Something like `<TestsRuntime>true</TestsRuntime>` or `<TestsCoreLib>true</TestsCoreLib>`.

Reply:
Good idea, I'll give it a shot this morning.